src/Mvc20xMediaBlock.cpp

```diff
@@ -2355,7 +2390,16 @@ namespace MediaBlock
             */
             if (state == mvc2::PlaybackControl::Running
                 || state == mvc2::PlaybackControl::Paused)
+            {
+                /* Check if start frame position has changed (in case seek performed) */
+                if (_startPos != _prev_start_pos)
+                {
+                    _startPos = _startPos - timeStamp / 90000.0;

```

Comment:
```suggestion
                    _startPos -= timeStamp / 90000.0;
```

Create a constant for `90000.0`

Reply:
Created.